### DISCUSSION TITLE
How do I only load certain data into a resource

### DISCUSSION THREAD

**Discussion by jcc5018** [filamentphp/filament#9794] [Help]
I have several items that utilize the same tables, but that I would like to have in separate pages. One of which is the tags/ taxonomy tables. 

I want to be able to have a link to only the categories or tags associated with a particular model.  Since there is no way that I know of to create links manually, the only current solution I see would be creating 60 copies of my tags resource which would violate DRY principles. 

How do I create a series of links (preferably nested under the relevant parent) such as in wordpress that would all point to the same resource but with a ->where('tag_type',#)   IE under my hobbies model i want hobby tags and categories, and then articles have their own tags, and so forth.

**Comment by PseudoDevs** [comment]
Hello, have you even try the Relation Manager?
You can refer to this link [https://filamentphp.com/docs/3.x/panels/resources/relation-managers#creating-a-relation-manager](url)

**Comment by jcc5018** [comment]
I'm going to assume your hostile tone is due to a language barrier and I will ignore it. 

Second, the link you sent is invalid but I found what I believe may be what you wanted to send.

Third. No I did not try a relationship manager because I'm exactly trying to establish a relationship. 

I want a link.../admin/hobby/categories,  which loads tags::where('tag_type','1' )and another admin/hobby/tags 
 (tags::where('tag_type','2)  article/tags... etc.  And so forth for all 60 tag types. But I have no idea where to include the code or where to add all the required links.

Or I can utilize the relation to display links for all tag types for each model. Ie my hobby model has 3 associated taxonomies, users has about 6.

I scanned through the relation manager and do not see any clear example of establishing that goal. 

Perhaps cause the docs don't have any real example to explain its concepts. So if you know specifically what I am looking for, I'd like a little more explanation than "read the docs" please. 

Cause docs often are not super clear for people new to a technology.

**Comment by ShaungBhone** [comment]
Did you mean that https://filamentphp.com/docs/3.x/panels/resources/getting-started#customizing-the-resource-eloquent-query?
